wemake_python_styleguide/logics/variables.py

```diff
@@ -40,6 +40,28 @@ def is_wrong_variable_name(name: str, to_check: Iterable[str]) -> bool:
     return False
 
 
+def is_upper_case_name(name: str):
+    """
+    Checks that attribute name has no upper-case letters.
+
+    >>> is_upper_case_name('camelCase')
+    True
```

Comment:
We utilize a single empty line between doctest examples.

Reply:
Will fix it.